prow/tide/codereview.go

```diff
@@ -0,0 +1,96 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package tide
+
+import (
+	"time"
+
+	"github.com/sirupsen/logrus"
+)
+
+type CodeReviewCommon struct {
+	// NameWithOwner is from graphql.NameWithOwner, <org>/<repo>
+	NameWithOwner string
+	// The number of PR
+	Number int
+	Org    string
+	Repo   string
+	// BaseRefPrefix gets prefix of ref, such as /refs/head, /refs/tags
+	BaseRefPrefix string
```

Comment:
This seems to be unused

Reply:
https://github.com/kubernetes/test-infra/blob/b0c002975451374244eb4b8a5ec1aec965bc98f4/prow/tide/tide.go#L1869 it's actually used